source/nodejs/adaptivecards-designer/src/app.ts

```diff
@@ -338,13 +360,14 @@ class DesignerApp {
         }
     
         this._selectedHostContainer.initialize();
-    
+
         this._designerHostElement.innerHTML = "";
         this._selectedHostContainer.renderTo(this._designerHostElement);
 
         this._designer = new Designer.CardDesigner(this._selectedHostContainer.cardHost);
         this._designer.onSelectedPeerChanged = (peer: Designer.CardElementPeer) => {
             this.buildPropertySheet(peer);
+            this.buildTreeViewSheet(peer);
```

Comment:
Not necessary here. You need only call this in the `this._designer.onLayoutUpdated` handler.